From 05f7d5796392f03c5f129e692eaafb9567ea8042 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 12 Apr 2023 08:03:04 +0000
Subject: [PATCH 09/23] [compiler-rt] [test] Pass the mingw specific option for
 increasing stack size

This passes the same option that was added for MSVC builds in
fb5a24b4f03a7 in the corresponding mingw form too.

This fixes the BVGraph tests in the sanitizer unit tests.
---
 compiler-rt/lib/sanitizer_common/tests/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/compiler-rt/lib/sanitizer_common/tests/CMakeLists.txt b/compiler-rt/lib/sanitizer_common/tests/CMakeLists.txt
index 2ff6d46ee10e..22bad618a8ff 100644
--- a/compiler-rt/lib/sanitizer_common/tests/CMakeLists.txt
+++ b/compiler-rt/lib/sanitizer_common/tests/CMakeLists.txt
@@ -113,6 +113,8 @@ endif()
 # The default stack size for clang is 8M.
 if(MSVC)
   list(APPEND SANITIZER_TEST_LINK_FLAGS_COMMON -Wl,/STACK:0xC00000)
+elseif (MINGW)
+  list(APPEND SANITIZER_TEST_LINK_FLAGS_COMMON -Wl,--stack=0xC00000)
 endif()
 
 set(SANITIZER_TEST_LINK_LIBS)
-- 
2.34.1

